frontend/src/css/styles.css

```diff
@@ -166,15 +165,25 @@ main .spinner .bounce2 {
 #previewer header {
   background: none;
   color: #fff;
+  border-bottom: 0px;
+  box-shadow: 0px 0px 0px;
+  z-index: 19999;
```

Comment:
This z-index will show the headerbar above the preview box. Preview box's z-index is 9999.
And the border-bottom, box-shadow will show a tiny line in the bottom of headerbar. So just erase it.